packages/components/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "fluent-react-components",
+  "version": "0.1.2",
+  "description": "Utility components for fluent-react",
+  "main": "./lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "test": "npm run lint && npm run cover",
+    "test:prod": "cross-env BABEL_ENV=production npm run test",
+    "test:only": "mocha --require babel-core/register --require babel-polyfill --recursive",
+    "cover": "istanbul cover _mocha -- --require babel-core/register --require babel-polyfill --recursive",
+    "lint": "eslint src test",
+    "build": "cross-env BABEL_ENV=production babel src --out-dir lib",
+    "prepublish": "npm run clean && npm run test && npm run build"
+  },
+  "keywords": [
+    "udacity",
+    "fluent",
+    "localization",
+    "l10n",
+    "ftl",
+    "translate",
+    "translation",
+    "fluent-react",
+    "internationalization",
+    "i18n",
+    "components",
+    "react"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/udacity/fluent-react-utils"
+  },
+  "files": [
+    "lib",
+    "src"
+  ],
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.1",
+    "babel-plugin-add-module-exports": "^0.2.1",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-minify": "^0.3.0",
+    "babel-preset-react": "^6.24.1",
+    "chai": "^4.1.2",
```

Comment:
Is there any particular reason for using chai and mocha for testing? Why not jest?

Reply:
https://github.com/udacity/fluent-react-utils/issues/19